resources/conf/config.toml

```diff
@@ -18,6 +18,7 @@
   listenerPort = 9090
   securedListenerPort = 9095
   systemHost = "localhost"
+  useRemoteAddress = false
```

Comment:
Shall we add this to the k8s artifact as well?

Reply:
Config added to `config-toml-configmap.yaml`